### PR TITLE
Clarify requirements for resolution of inconsistent information (Paul Wouters' IESG review)

### PR DESCRIPTION
**Pull request by cabo**

### CODE REVIEW

File: draft-ietf-sedate-datetime-extended.md [hunk 1]
```diff
@@ -789,7 +789,8 @@ extensions still to be defined.
 Where consistent interpretation between multiple actors is required
 for security purposes (e.g., where timestamps are embedded as
 parameters in access control information), only such extensions can be
-employed that have a defined resolution of such inconsistent data.
+employed that have a well-understood and shared resolution of such
```
Comment (ediosyncratic) [inline draft-ietf-sedate-datetime-extended.md:792]: Grammar: s/that/as/.

That's "only such [whatever] as have [what's needed]"; contrast with "you can use [a thing] that has [what's needed], but not one that lacks it."

**Review by ryzokuken — APPROVED** [review]

File: draft-ietf-sedate-datetime-extended.md (hunk 1 above)
Comment (ryzokuken) [reply draft-ietf-sedate-datetime-extended.md:792]: Don't feel too strongly about this either way, but that feels better here.

Perhaps `s/that/which` ("...only such extensions can be employed which have a...")

**Review by ryzokuken — APPROVED** [review]
those does read better here

File: draft-ietf-sedate-datetime-extended.md (hunk 1 above)
Comment (ediosyncratic) [reply draft-ietf-sedate-datetime-extended.md:792]: (Well, at this point, I'd s/that/which/ but it'll do.)

**Pull request merged**